modules/ROOT/partials/commands/advtable-cmds.adoc

```diff
@@ -36,6 +36,7 @@ tinymce.activeEditor.execCommand('mceSortTableByRowDesc')
 tinymce.activeEditor.execCommand('mceTableToggleSeries', false, { name: 'numeric' })
 ----
 
+[[usingmcesorttableadvanced]]
```

Comment:
I can't leave a comment on the right line sorry, but in the above there's a broken `include::` that's being rendered verbatim.

Reply:
done